2017/07/04/a-reason-react-tutorial/index.html

```diff
@@ -348,7 +348,7 @@ <h3 id="16-the-todoitem-component">The TodoItem component</h3>
 </iframe></div>
     </div>
 
-<p>So this is another stateless component, except this one accepts a property: <code>item</code>. The <code>::argname</code> syntax means &quot;this function takes a labeled argument which is known as <code>text</code> both externally and internally&quot;. Swift and Objective C also allow you have labeled arguments, with an external name that is optionally different from the internal one. If you wanted them to be different, you would write e.g. <code>fun externalFacingName::internalFacingName =&gt;</code>. <code>children</code> is an unnamed argument.</p>
+<p>So this is another stateless component, except this one accepts a property: <code>item</code>. The <code>::item</code> syntax means &quot;this function takes a labeled argument which is known as <code>item</code> both externally and internally&quot;. Swift and Objective C also allow you have labeled arguments, with an external name that is optionally different from the internal one. If you wanted them to be different, you would write e.g. <code>fun externalFacingName::internalFacingName =&gt;</code>. <code>children</code> is an unnamed argument.</p>
```

Comment:
This changed:

```
which is known as <code>text</code>
```

to

```
which is known as <code>item</code>
```

Since the argument name is `item`